compiler-plugin/src/main/kotlin/com/google/devtools/ksp/Incremental.kt

```diff
@@ -288,6 +303,17 @@ class IncrementalContext(
         return visited
     }
 
+    private fun logDirtyFilesByCP(dirtyFiles: Collection<File>) {
+        if (!options.incrementalLog)
+            return
+
+        val logFile = File(logsDir, "kspDirtySetByCP.log")
```

Comment:
Is this file annotated as task output? If so, it may be an issue if cache is relocatable i.e output from one machine is retrieve on another machine.

Reply:
Logs are not annotated at all; They are generated into `build/`.

Reply:
Is it unique per-task? Also I feel that moving them to the incremental dir (which is annotated with @LocalState) is the way to go. In such way:
- it is not restored from cache when there is a cache hit
- in a non-incremental run you already have logic to clean up the entire dir, so things are consistent and correct
wdyt?

Reply:
Done: c397a30